### PR TITLE
[aarch64] build torch library with mkldnn+acl backend

### PR DESCRIPTION
**Pull request by snadampal**

### CODE REVIEW

**Comment by snadampal** [comment]
@JackCaoG , could you please review this? context: This change was left from the [PR](https://github.com/pytorch/xla/pull/3660) where I was adding the changes incrementally.

**Comment by snadampal** [comment]
Hi @JackCaoG , @miladm , @will-cromar  please let me  know if there is any feedback on this pr.

File: scripts/build_torch_wheels.sh [hunk 1]
```diff
@@ -76,7 +78,14 @@ function maybe_install_cuda {
...
 function maybe_install_sources {
-  if [ ! -d "torch" ]; then
+  if [[ $(uname -m) == "aarch64" && ! -d "$HOME/ComputeLibrary" ]]; then
+    # install arm compute library
+    pushd $HOME
+    git clone https://review.mlplatform.org/ml/ComputeLibrary.git
+    popd
+  fi
+
+  if [ ! -d "pytorch" ]; then
```
[the remark this answers is not in the document]
Comment (JackCaoG) [reply scripts/build_torch_wheels.sh:88]: no idea.. I think `torch` also works but `pytorch` is more precise I guess. If things start to fall apart we can revert this part too

**Comment by JackCaoG** [comment]
Mostly LGTM, @snadampal I assumed you have tested this script on your end?

**Comment by snadampal** [comment]
Yes, I have tested it on aarch64 and x86 as well.

**Comment by snadampal** [comment]
Hi @JackCaoG , if there are no comments, could you please merge this PR?

**Review by JackCaoG — APPROVED** [review]

**Comment by snadampal** [comment]
@JackCaoG Thanks for merging the pr!

**Pull request merged**